acceptance/ui/features/user.feature

```diff
@@ -0,0 +1,45 @@
+@user
+Feature: User Details
+  As a Control Center user
+  I want to manage user details
+
+  @login-required
+  Scenario: View user details
+    When I view user details
+    Then I should see "User Details"
+      And I should see "Username: zenoss"
+      And I should see "Messages"
+      And I should see "Clear"
+
+  @login-required @defaultHostPage
+  Scenario: View all messages
+    When I view user details
+    Then I should see my messages
+
+  @login-required @defaultHostPage
+  Scenario: Mark messages as read
+    When I view user details
+      And I click on an unread message
+    Then I should see a checkmark
```

Comment:
Let's discuss this one. Ideally, the test would verify that the message clicked on was the message marked as read, but given we don't know what messages will be there, that's a little hard to verify